packages/lit-element/src/lib/lit-element.ts

```diff
@@ -126,58 +125,42 @@ export class LitElement extends UpdatingElement {
    * using the [[`css`]] tag function or via constructible stylesheets.
    */
   static styles?: CSSResultOrNative | CSSResultArray;
-
-  private static _styles: Array<CSSResultOrNative | CSSResult> | undefined;
+  private static elementStyles?: CSSResultArray;
```

Comment:
Name private variables with an underscore? Or double underscore so we can minify it?

Reply:
Done.